frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/artifacts/ArtifactNodeDetails.tsx

```diff
@@ -90,14 +88,7 @@ export const ArtifactNodeDetails: React.FC<ArtifactNodeDetailsProps> = ({
             <DescriptionListGroup>
               <DescriptionListTerm>{artifactName}</DescriptionListTerm>
               <DescriptionListDescription>
-                <Stack hasGutter>
-                  <StackItem>
-                    <ArtifactUriLink uri={artifact.getUri()} />
-                  </StackItem>
-                  <StackItem>
-                    <ArtifactPreview artifact={artifact} />
-                  </StackItem>
-                </Stack>
+                <ArtifactUriLink uri={artifact.getUri()} />
```

Comment:
I'm not following how these changes are relevant to the issues being addressed in this PR.

Reply:
Thank you @jeff-phillips-18 rebased